### PR TITLE
Add Apps UI example for tx format

### LINKED ISSUE #1172: TODO in https://docs.substrate.io/reference/transaction-format/
### Is there an existing issue?

- [X] I have searched the existing issues

### Experiencing problems? Have you tried our Stack Exchange first?

- [X] This is not a support question.

### What browsers are you seeing the problem on?

Chrome

### Bug report for the website

There is a pending TODO in https://docs.substrate.io/reference/transaction-format/

`[ TODO: polkadotjs apps screenshot ]`

### Steps to reproduce the problem

Go to https://docs.substrate.io/reference/transaction-format/

### PR DESCRIPTION
**Pull request by NukeManDan**
closes #1172 with added depth and screenshots for reference.

### CODE REVIEW

File: content/md/en/docs/reference/transaction-format.md [hunk 1]
```diff
@@ -84,24 +84,38 @@ The way applications know how to construct a transaction correctly is provided b
 An application will know how to correctly encode a transaction by using the metadata types and transaction format.
 If a call doesn't need to be signed, then the first bit in `[2]` will be 0 and so an application will know not to try decoding a signature.
 
-**Example:**
+**Polkadot JS Apps example:**
 
-Balances transfer from Bob to Dave: Bob sends `42` units to Dave.
+Here we demonstrate a detailed, manual extrinsic construction an submission of a balance transfer from Bob to Dave: Bob sends `42 UNIT`s to Dave.
```
Comment (IkerAlus) [inline content/md/en/docs/reference/transaction-format.md:89]:
```suggestion
Here we demonstrate a detailed, manual extrinsic construction and submission of a balance transfer from Bob to Dave: Bob sends `42 UNIT`s to Dave.
```

File: content/md/en/docs/reference/transaction-format.md [hunk 2]
```diff
@@ -84,24 +84,38 @@ The way applications know how to construct a transaction correctly is provided b
...
 
-[ TODO: polkadotjs apps screenshot ]
+1. Start a [node template](https://github.com/substrate-developer-hub/substrate-node-template) in `--dev` mode (see the [quick start](/quick-start) guide for how to set this up)
+1. Navigate to <https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/extrinsics>
+1. Set `Bob` as sender and select `balances` pallet and it's `transfer(dest, value)` call
```
Comment (IkerAlus) [inline content/md/en/docs/reference/transaction-format.md:93]: shouldn't this be "...the `transfer(dest, value)` call"?

**Review by IkerAlus — CHANGES_REQUESTED** [review]
Only a couple of minor issues. The rest LGTM.

File: content/md/en/docs/reference/transaction-format.md (hunk 2 above)
Comment (NukeManDan) [reply content/md/en/docs/reference/transaction-format.md:93]: open to a direct suggestion @IkerAlus  - use `ctrl+g` to pull that up in a comment here ;)

Comment (NukeManDan) [reply content/md/en/docs/reference/transaction-format.md:93]:
```suggestion
1. Set `Bob` as sender and select `balances` pallet and the `transfer(dest, value)` call
```

**Review by IkerAlus — APPROVED** [review]
LGTM

**Pull request merged**